index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name='viewport' content='width=device-width, initial-scale=1, maximum-scale=1'>
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.scss">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header__logo header-logo">
+            <img class="header-logo__logo-image" src="images/askit-logo.png" />
+        </div>
+        <nav class="header__nav-panel header-nav-panel">
+            <ul class="header-nav-panel__menu">
+                <li class="header-nav-panel__menu-item"><a href="#">Home</a></li>
+                <li class="header-nav-panel__menu-item"><a href="#">How it works</a></li>
+                <li class="header-nav-panel__menu-item"><a href="#">Features</a></li>
+                <li class="header-nav-panel__menu-item"><a href="#">Partners</a></li>
+                <li class="header-nav-panel__menu-item"><a href="#">Events</a></li>
+                <li class="header-nav-panel__menu-item"><a href="#">About us</a></li>
+            </ul>
+        </nav>
+        <div class="header__authentication header-authentication">
+            <div class="header-authentication__log-in header-log-in">
+                <img class="header-log-in__image" src="images/menu-sign-in.png" />
+                <p>Log in</p>
+            </div>
+            <div class="header-authentication__register header-register">
+                <img class="header-register__image" src="images/menu-avatar.png" />
+                <p>Register</p>
+            </div>
+        </div>
+
+        <div class="header__menu-burger header-menu-burger">
+            <img class="header-menu-burger__image" src="images/mobile-burger-menu.png" />
+        </div>
+    </header>
+    <section class="live-interaction">
+        <div class="live-interaction__summary live-interaction-summary">
+            <h1 class="live-interaction-summary__header">Live Interaction From Audience</h1>
+            <div class="live-interaction-summary__borderline">
+                <div class="live-interaction-summary__borderline-small"></div>
+                <div class="live-interaction-summary__borderline-middle"></div>
+                <div class="live-interaction-summary__borderline-big"></div>
+            </div>
+            <p class="live-interaction-summary__text">Q&A, Polls and Slide-sharing for Events</p>
+
+            <div class="live-interaction__btns-list live-interaction-btns-list">
```

Comment:
**live-interaction__btns-list** line means this block is element of **live-interaction** component but this component should be element of **live-interaction-summary**